src/entities/bond.ts

```diff
@@ -64,33 +69,44 @@ export class Bond {
         return BigNumber.from(this.data.totalDebt);
     }
 
+    get totalDebtAtMaturity(): BigNumber {
+        return BigNumber.from(this.data.totalDebtAtMaturity);
+    }
+
     get totalCollateral(): BigNumber {
+        return BigNumber.from(this.data.totalCollateral);
```

Comment:
Would it make sense to merge this and `totalCollateralAtMaturity` and just return the latter if mature? totalCollateral is always 0 after maturity anyways

Reply:
I was in two minds about that, originally had them merged, but there's places where you need to retain the distinction
(eg. calculating tranche redemption amounts, you need to use live values since the collateral is redeemed and rebased)

So I decided to keep it most consistent and have these split out here too.